modules/contact-form/admin.php

```diff
@@ -902,3 +940,65 @@ function grunion_recheck_queue() {
 }
 
 add_action( 'wp_ajax_grunion_recheck_queue', 'grunion_recheck_queue' );
+
+/**
+ * Delete a number of spam feedbacks via an AJAX request.
+ */
+function grunion_delete_spam_feedbacks() {
+	if ( ! wp_verify_nonce( $_POST['nonce'], 'jetpack_delete_spam_feedbacks' ) ) {
+		wp_send_json(
+			array(
+				'error' => __( 'You don&#8217;t have permission to do that.', 'jetpack' ),
+			)
+		);
+
+		return;
+	}
+
```

Comment:
Should we check for capabilities here, before we go any further and query for feedback posts?

Reply:
Do you think it's reasonable to replace the `current_user_can( 'delete_post', $feedback->ID )` calls below with a single `current_user_can( 'delete_posts' )` here?  Or maybe `delete_others_posts` is more appropriate?

Reply:
Yeah, that would make sense and simplify things even more 👍. `delete_others_posts` seems fine here.

Reply:
I made the change in a786416b4.